app/src/vanilla/java/cm/aptoide/pt/view/wizard/WizardPageOneFragment.java

```diff
@@ -21,6 +21,11 @@ public static Fragment newInstance() {
     return new WizardPageOneFragment();
   }
 
+  @Override public void onCreate(@Nullable Bundle savedInstanceState) {
```

Comment:
on SettingsFragment.java the view is registered on onResume method, should this one be registered here?